### ISSUE TITLE
"WARNING:tensorflow:Variable += will be deprecated"

### ISSUE THREAD

**Issue by prabindh** [keras-team/keras#13121]
**System information**  
- Have I written custom code (as opposed to using example directory):  No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Windows 10.0.16299.1217
- TensorFlow backend (yes / no):  yes
- TensorFlow version:  1.13.1
- Keras version:  2.2.4 to current
- Python version:  3.7.1
- CUDA/cuDNN version: NA 
- GPU model and memory:  NA

**Describe the current behavior**  
When running the example "neural_style_transfer.py", getting a warning as below:
"WARNING:tensorflow:Variable += will be deprecated"

**Describe the expected behavior**  
Deprecation warnings should be heeded and replaced with compliant code.

**Code to reproduce the issue**  
Use existing example - examples/neural_style_transfer.py

**Other info / logs**  
A typical output when running neural_style_transfer is below:
Using TensorFlow backend.
<>WARNING:tensorflow:Variable += will be deprecated. Use variable.assign_add if you want assignment to the variable value or 'x = x + y' if you want a new python Tensor object.

**Comment by jvishnuvardhan** [comment]
@prabindh Warnings are due to tensorflow which is going to release 2.0 (Huge shift from TF1.x) and deprecate TF1.x related API's in the future. You could disable those warning by using simple code given below. Add the following code at the top of your code to disable warning or info etc. Thanks!

```
import os
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'  # or any {'0', '1', '2'}
# 0 (all logs shown), 
# 1 to filter out INFO logs, 
# 2 to additionally filter out WARNING logs, and
# 3 to additionally filter out ERROR logs
```

**Comment by prabindh** [comment]
Thanks. Agreed we could filter out all warnings, but then all other warnings would be masked, which isnt the right thing in my opinion. What stops us from moving to like below commit,

https://github.com/prabindh/keras/commit/063bfcbf43c0e315e807944caeeed07a44a0d33f

**Comment by prabindh** [comment]
Any update ?
